src/components/childComponents/table/CgeResultsTable.jsx

```diff
@@ -130,6 +137,17 @@ export function CgeResultsTable(props) {
 							}
 						</TableRow>
 
+						<TableRow>
+							<TableCell className={classes.tableRowHead} style={{textAlign:"right"}}> % of Change </TableCell>
+							{
+								headers.slice(1).map( (header, idx) => {
+									return (<TableCell className={classes.tableDataCell} key={idx}>
+										{(percentageChange[header] !== undefined ? `${round(percentageChange[header], 1)} %`: "-" )}
```

Comment:
not sure it is related to this particular line, but there are multiple duplicate key errors in the console. They happen on every tab change

Reply:
I fixed this one. But i still see duplicate key error message in the ViewResultEntry table. Will open a more generic issue to address all the warning/error seeing in the console?

Reply:
https://github.com/IN-CORE/incore-playbook/issues/21